input/otlp/traces.go

```diff
@@ -325,7 +331,7 @@ func TranslateTransaction(
 				if ip, err := modelpb.ParseIP(stringval); err == nil {
 					event.Source.Ip = ip
 				}
-			case semconv.AttributeNetPeerName:
+			case semconv.AttributeNetPeerName, attributeServerAddress:
```

Comment:
As above, I don't think it's appropriate to set `event.source.domain = server.address`